mscore/inspector/inspector_tempotext.ui

```diff
@@ -250,16 +250,14 @@
   <customwidget>
    <class>Ms::ResetButton</class>
    <extends>QWidget</extends>
-   <header>inspector/resetButton.h</header>
+   <header location="global">inspector/resetButton.h</header>
    <container>1</container>
   </customwidget>
  </customwidgets>
  <tabstops>
   <tabstop>title</tabstop>
   <tabstop>followText</tabstop>
   <tabstop>tempo</tabstop>
-  <tabstop>style</tabstop>
```

Comment:
Please add the tabstops back by entering "edit tabs" mode in Qt Creator/Designer. Except that now they should be in front of the tabstops of `followText`.

Reply:
esp. important for accessibility!